test/integration/framework/etcd.go

```diff
@@ -136,35 +138,89 @@ func RunCustomEtcd(dataDir string, customFlags []string, output io.Writer) (url
 	}
 	args = append(args, customFlags...)
 	cmd := exec.CommandContext(ctx, etcdPath, args...)
-	if output == nil {
-		cmd.Stdout = os.Stdout
-		cmd.Stderr = os.Stderr
-	} else {
-		cmd.Stdout = output
-		cmd.Stderr = output
+
+	// Always filter the etcd output. This allows us to get rid of known harmless messages and (more importantly)
+	// to detect the "listen tcp 127.0.0.1:37803: bind: address already in use" error.
```

Comment:
where is this error coming?

Reply:
This comment is obsolete. The remaining purpose of output interception is making the output nicer, not detecting certain events.